rdrf/rdrf/templatetags/get_info_text_expression.py

```diff
@@ -9,22 +8,25 @@
 
 
 @register.filter()
-def get_information_text(fields):
+def get_info_text_expression(fields):
```

Comment:
Minor point coming to this cold I found the method name a little misleading. It gets the path (URI) to a template rather than getting info text.

Also, a bit worried that this abstracts too much. It might be better if the URI was in the actual template and not hidden behind the filter. So the template code can be something like:

`{% include "rdrf://model/ConsentSection/%s/information_text" with consent_section=get_consent_section %}
`
(Forgive me if I munged the django template syntax)